css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,21 @@
+.custom-button {
+  color: blue;
+  cursor: pointer;
+}
+
+.custom-button:hover {
+  background-color: orange;
+}
+
+thead {
+  color: rgb(255 255 255);
+  background-color: black;
+}
+
+tr.even {
+  background-color: #9d00ff;
+}
+
+tr.odd {
+  background-color: yellow;
+}
```

Comment:
Please accomplish this with pseudo-classes rather than applying classes manually yourself.